theseus/src/api/jre.rs

```diff
@@ -190,3 +190,9 @@ pub async fn get_max_memory() -> crate::Result<u64> {
         })?
         .total)
 }
+
+pub async fn extract_version_from_string(
+    version: &str,
+) -> crate::Result<(u32, u32)> {
+    Ok(extract_java_majorminor_version(version)?)
+}
```

Comment:
what is this func used for?

Reply:
Don't think this is neccessary since the correct java version checking should be in the backend not the frontend